edm4eic.yaml

```
@@ -614,6 +614,32 @@ datatypes:
       - edm4eic::Cluster  cluster           // reference to the cluster
       - edm4eic::Track track                // reference to the track
 
+  ## ==========================================================================
+  ## Data-Monte Carlo comparisons
+  ## ==========================================================================
+
+  edm4eic::Truthiness:
+    Description: "Positive-definite convex norm of how confidently wrong the reconstruction is,
+                  with non-negative contributions from various aspects of the reconstruction,
+                  where a zero value indicates a perfect reconstruction."
+    Author: "W. Deconinck, S. Colbert"
+    Members:
+       - float event_truthiness                       // Overall truthiness of the entire event
+       - float event_pid_contribution                 // Contribution of PID matching to truthiness
+       - float event_energy_contribution              // Contribution of energy matching to truthiness
+       - float event_momentum_contribution            // Contribution of momentum matching to truthiness
+       - float unassociated_mc_particles_contribution // Contribution from unassociated MC particles
+       - float unassociated_rc_particles_contribution // Contribution from unassociated reconstructed particles
```

For consistency with other types, I'd suggest using camel case for field names.
```suggestion
       - float eventTruthiness                       // Overall truthiness of the entire event
       - float eventPIDContribution                 // Contribution of PID matching to truthiness
       - float eventEnergyContribution              // Contribution of energy matching to truthiness
       - float eventMomentumContribution            // Contribution of momentum matching to truthiness
       - float unassociatedMCParticlesContribution // Contribution from unassociated MC particles
       - float unassociatedRecoParticlesContribution // Contribution from unassociated reconstructed particles
```